### DISCUSSION TITLE
Is it possible to configure ov to open an editor with a key binding?

### DISCUSSION THREAD

**Discussion by ryuheechul** [noborus/ov#267] [Q&A] [answered]
For example what I would like to have is that when run `ov` with a file it works like `less` but when I type `e` at any moment it opens with $EDITOR. Is there a way to configure `ov` to work like that?

**Comment by noborus** [comment, marked as answer]
Good question.
`ov` does not yet have the ability to open an editor.
However, if you do the following, it will work in the same way, so I think it will be possible to implement it in the future.

1. suspend
2. Start the editor.
3. resume
4. reload

However, if you want the equivalent of `LESSEDIT`, things get a little more complicated.

**Reply by ryuheechul** [reply]
I like your idea idea of suspend edit resume reload. And I didn’t want `LESSEDIT` as I don’t even understand it clearly.

Simply pager delegating to editor until editor quits would be great.

**Reply by noborus** [reply]
Thank you for your reply.
I don't use LESSEDIT either, but I'll try to implement it if there's demand for it.
